java-container/google-cloud-container/src/test/java/com/google/cloud/container/v1/it/ITSystemTest.java

```diff
@@ -90,7 +91,13 @@ public static void beforeClass() throws Exception {
 
   @AfterClass
   public static void afterClass() throws Exception {
-    Thread.sleep(TimeUnit.MINUTES.toMillis(5));
+    Operation response = client.getOperation(PROJECT_ID, ZONE, operation.getName());
+    // Sleep for one minute until Cluster CREATE operation is complete
```

Comment:
Why are we waiting for cluster creation completion in `afterClass()`? Shouldn't we wait for it right after creation in `beforeClass()`?

Reply:
I was able to run operations of the client (get/ list) without the operation having to be complete. It was only when deleting the cluster, I got an error with `Operation in progress`. I think it would make sense to move this to wait when the operation was run.

I'll change this to to do the busy wait in beforeClass().